source/fundamentals/utf8-validation.txt

```diff
@@ -0,0 +1,121 @@
+.. _nodejs-utf-8-validation:
+
+================
+UTF-8 Validation
+================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to enable or disable the {+driver-short+}'s
+**UTF-8** validation feature. UTF-8 is a character encoding specification
+that ensures compatibility and consistent presentation across most operating
+systems, applications, and language character sets.
+
+If you *enable* validation, the driver throws an error when it attempts to
+convert data that contains invalid UTF-8 characters. The validation adds
+processing overhead since it needs to check the data.
+
+If you *disable* validation, your application avoids the validation processing
+overhead, but cannot guarantee consistent presentation of invalid UTF-8 data.
+
+The driver enables UTF-8 validation by default. It checks documents for any
+characters that are not encoded in a valid UTF-8 format when it transfers data
+between your application. If it encounters invalid UTF-8 characters, it
```

Comment:
[issue/ suggestion]

"... transfers data between your application and MongoDB", or

"... transfers data from MongoDB to your application" (read only?)

Reply:
Great catch, I missed that while rewriting it. It's bi-directional (serialization and deserialization), so between works best.